src/training/shield.rs

```diff
@@ -145,13 +145,33 @@ fn handle_shield_decay(param_type: u64, param_hash: u64) -> Option<f32> {
     None
 }
 
+/// This is the cached shield damage multiplier.
+/// Vanilla is 1.19, but mods can change this.
+static mut CACHED_SHIELD_DAMAGE_MUL: f32 = -1.0;
```

Comment:
For these cases, `Option<f32> = None`?

Reply:
significantly better way to express, thank you. Thought about using like a OnceCell or something but decided it wasn't gonna be a big deal really.